sympy/geometry/line.py

```diff
@@ -1504,8 +1642,9 @@ def plot_interval(self, parameter='t'):
         Returns
         =======
 
-        plot_interval : list
-            [parameter, lower_bound, upper_bound]
+        list :
```

Comment:
```suggestion
        plot_interval : list
```

Reply:
I think I have messed up many things in one commit. I will make changed to this module and will generate a pull request after that.
Thanks for the help.